integration_tests/ci/sample.profiles.yml

```diff
@@ -11,7 +11,7 @@ integration_tests:
   outputs:
     postgres:
       type: postgres
-      host: localhost
+      host: "{{ env_var('CI_DBT_HOST') }}"
```

Comment:
This would mean btw that this environment variable would need to be set in CircleCI too, or we should put `localhost` as a default value, but that would not be my preference.

Now that I take another look on it, adding `POSTGRES` somewhere in these environment variables, e.g. `CI_POSTGRES_DBT_HOST` would probably make the naming more consistent in the profiles.yml